tox.ini

```
@@ -26,7 +26,7 @@ deps =
     pytest-verbose-parametrize==1.4.0
     pytest-xdist==1.26.0
     shade==1.22.2
-    yapf>=0.25.0,<2
+    yapf>=0.25.0,<0.27
```

https://github.com/ansible/molecule/issues/1955 add filtering as per doc mentioned here

---

I'd normally have no problem with it, but don't have the cycles to learn a new tool today. I assume it's simple, but I'm probably not going to get a chance to look at the pyup docs til next week.

---

Alright, just hit apply here:
```suggestion
    yapf>=0.25.0,<0.27  # pyup: < 0.27 # disable updates, conflicts with flake8 as per https://github.com/ansible/molecule/pull/1915
```

---

Oh, that's perfect, thanks!

---

There are two places where this dep is declared in the file (with good reason, imo), so rather than hit the button I'm going to steal and give credit.

---

@seandst Yeah... Well, the difference is that GitHub recognizes `Co-Authored-By: User <email>` trailer in commits which in this case you should add manually. Otherwise, GitHub doesn't have metadata for a co-author.